loopd/swapclient_server.go

```diff
@@ -234,6 +235,42 @@ func (s *swapClientServer) Monitor(in *looprpc.MonitorRequest,
 	}
 }
 
+// GetSwaps returns a list of all currently known swaps and their current
+// status.
+func (s *swapClientServer) GetSwaps(_ context.Context,
+	_ *looprpc.GetSwapsRequest) (*looprpc.GetSwapsResponse, error) {
+
+	var (
+		rpcSwaps = make([]*looprpc.SwapStatus, len(swaps))
+		idx      = 0
+		err      error
+	)
+	for _, swp := range swaps {
```

Comment:
Any particular reason you opted to use a new index rather than the index from the range loop here?

Reply:
Yes, the `swaps` variable is a map.